manoseimas/mps_v2/static/components/filter_component.jsx

```diff
@@ -9,6 +9,7 @@ var SortableList = React.createClass({
       sort_order: this.props.default_order,
       filter_selected: 'all',
       filter_options: null,
+      subtab_selected: this.props.active_subtab || 'lobbyists',
```

Comment:
This feels like a generic class here that shouldn't know about "lobbyists" or anything.